docs/customization/model.rst

```diff
@@ -399,11 +449,20 @@ Which we strongly recommend over updating the schema.
     $ php bin/console doctrine:migrations:diff
     $ php bin/console doctrine:migrations:migrate
 
+.. warning::
+
+    Sometimes when something goes wrong and you get database-errors you must use ``doctrine:database:drop`` but **remember this command destroy you database !!!**
+
+        .. code-block:: bash
```

Comment:
I think that these spaces between the beginning of the line and `.. code-block:: bash` are not necessary and cause this block to render incorrectly

Reply:
You wrong. This is ok

Reply:
Are you sure? Take a look at [this](https://github.com/Sylius/Sylius/blob/master/docs/customization/controller.rst#L50) and [this](https://github.com/Sylius/Sylius/blob/master/docs/customization/model.rst#L46) 🖖